skypy/galaxy/spectrum.py

```diff
@@ -197,6 +198,77 @@ def mag_ab(spectrum, bandpass, redshift=None):
     return mag_ab.item() if not return_shape else mag_ab.reshape(return_shape)
 
 
```

Comment:
This function could be decorated with `@spectral_data_input` so that keywords like `templates='kcorrect_spec', bandpass='DECam_griz'` would be accepted.